.github/workflows/deny.yml

```diff
@@ -0,0 +1,65 @@
+name: Deny - Linux
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    # At midnight UTC
+    - cron: '0 0 * * *'
+
+jobs:
+  test-misc:
+    runs-on: [linux, ubuntu-20.04-4core]
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I'm not seeing any reason this couldn't be `ubuntu-latest` and save on CI time? For that matter 🤔  can't recall why `misc` needs that.

```suggestion
    runs-on: ubuntu-latest
```

Reply:
Good call.